db/migrate/20170302151706_add_new_recipe_label_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddNewRecipeLabelToUsers < ActiveRecord::Migration[5.0]
+  def change
+    User.all.each do |user|
+      Label.create_defaults user
```

Comment:
Instead of calling `create_defaults`, I propose that we only create the label we have newly added. Reason being that if a user changes the name of one of the default labels, this method would create another label with the old name. But since I highly doubt, that any of our users changed the name of the defaults, this is good to do. 😄